docker/dev/supervisor.conf

```diff
@@ -0,0 +1,11 @@
+programs:
+  - name: "vpp"
+    executable-path: "/usr/bin/vpp"
+    executable-args: ["-c", "/etc/vpp/vpp.conf"]
+    logfile-path: "/var/log/supervisord.log"
```

Comment:
This is not correct log file path for vpp.

Reply:
The log file path can be whatever you want, so I don't think there is some "correct" setup. Also, it's the same path we have set before (just for reference, I agree to remove it as stated below)

Reply:
You might be confusing log of supervisord process:

https://github.com/ligato/vpp-agent/blob/80401e66a0ef370ac61c404590de51d86f78bf4c/docker/dev/supervisord.conf#L2

The output of both VPP and Agent were printed through supervisord stdout/stderr.